reference/configuration/doctrine.rst

```diff
@@ -499,4 +499,135 @@ can be placed directly under ``doctrine.orm`` config level.
 This shortened version is commonly used in other documentation sections.
 Keep in mind that you can't use both syntaxes at the same time.
 
+Custom Mapping
+--------------
+
+It's possible to not rely on the auto mapping and set all the configuration
+on your own.
+
+For example, you can overwrite the configuration directory for a given bundle.
+Keep in mind that the bundle you want to overwrite has to exist.
```

Comment:
How about something like this:

```rst
Doctrine's ```auto_mapping`` feature loads annotation configuration from the
``Entity/`` directory of each bundle *and* looks for other formats (e.g. YAML, XML)
in the ``Resources/config/doctrine``directory.

If you store metadata somewhere else, you can define your own mappings,
where you tell Doctrine exactly *where* to look, along with some other information.

For example, suppose you decide to store your configuration for `AppBundle` entities
in the ``app/Resources/config/doctrine`` directory instead
```